src/Element.php

```diff
@@ -195,28 +195,27 @@ public function setOption($key, $value)
      */
     public function setAttribute($key, $value)
     {
-        // Do not include the value in the list of attributes
-        if ($key === 'value') {
+        if ($key == 'value') {
+            // Do not include the value in the list of attributes
             $this->setValue($value);
-            return $this;
+        } else {
+            $this->attributes[$key] = $value;
         }
-        $this->attributes[$key] = $value;
         return $this;
     }
 
     /**
      * Retrieve a single element attribute
      *
-     * @param  $key
-     * @return mixed|null
+     * @param string $key
+     * @return mixed
```

Comment:
With the changes you are proposing, this should definitely be `null|mixed` for the return value, as `null` is what is returned if the attribute cannot be found.

Reply:
You "resolved" the conversation without reverting to the original as requested.